pkg/linter/linter.go

```diff
@@ -156,6 +161,40 @@ func worldWriteableLinter(_ LinterContext, path string, d fs.DirEntry) error {
 	return nil
 }
 
+func strippedLinter(lctx LinterContext, path string, d fs.DirEntry) error {
+	if !d.Type().IsRegular() {
+		// Don't worry about non-files
+		return nil
+	}
+
+	info, err := d.Info()
+	if err != nil {
+		return err
+	}
+
+	ext := filepath.Ext(path)
+	mode := info.Mode()
+	if mode&0111 == 0 && !isObjectFileRegex.MatchString(ext) {
+		// Not an executable or library
+		return nil
+	}
+
+	file, err := elf.Open(filepath.Join(lctx.realfspath, path))
+	if err != nil {
+		// We don't particularly care if this fails, it means it's probably not an ELF file
+		fmt.Printf("WARNING: Could not open file '%s' as executable: %v\n", path, err)
+		return nil
+	}
+	defer file.Close()
+
+	// No debug sections allowed
+	if file.Section(".debug") != nil || file.Section(".zdebug") != nil {
+		return fmt.Errorf("File is not stripped")
```

Comment:
It might be helpful for this error message to include the file path.

Reply:
It prints it with the error, see Line 254.